gradio/chat_interface.py

```diff
@@ -645,25 +716,22 @@ def _setup_events(self) -> None:
                 queue=False,
             )
 
-            @on(
-                [self.load, self.saved_conversations.change],
+            on(
+                triggers=[self.load, self.saved_conversations.change],
+                fn=self._load_chat_history,
                 inputs=[self.saved_conversations],
                 outputs=[self.chat_history_dataset],
                 show_api=False,
                 queue=False,
             )
-            def load_chat_history(conversations):
```

Comment:
Decided to avoid using decorator syntax for consistency with the rest of the file